src/OpenSage.Game/Client/Drawable.cs

```diff
@@ -623,5 +635,7 @@ public enum ObjectDecalType
         NationalismVehicle = 4, // exhordeb_up.dds
         Crate = 5, // exjunkcrate.dds
         None = 6,
+        Unknown1 = 7,
+        Unknown2 = 8,
```

Comment:
I also came across `Unknown2`, which an `AmericaVehicleDozer` had? I found this interesting since I'm not aware of any visible decal drawn for a USA dozer...